src/NosCore.GameObject/Character.cs

```diff
@@ -716,7 +719,7 @@ public void Buy(Shop shop, short slot, short amount)
             }
         }
 
-        private NInvPacket BuyFrom(ShopItem item, short amount, short slotChar)
+        private async Task<NInvPacket?> BuyFrom(ShopItem item, short amount, short slotChar)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'BuyFrom'.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5262615)